edgexfoundry/device-virtual-go
# refactor: consume device-sdk-go breaking changes

update for the new ProtocolDriver interface changes







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-virtual-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Run device-virtual from this branch
2. Verify the service is working correctly

## New Dependency Instructions (If applicable)
